lms/djangoapps/discussion_api/tests/test_views.py

```diff
@@ -851,15 +917,23 @@ def setUp(self):
         super(CommentViewSetCreateTest, self).setUp()
         self.url = reverse("comment-list")
 
-    def test_basic(self):
-        self.register_get_user_response(self.user)
+    def register_thread(self, closed=False):
```

Comment:
This is used in a similar way 3 times. Could it be added to DiscussionAPIViewTestMixin?